ofrak_type/setup.py

```diff
@@ -21,7 +21,7 @@ def run(self):
 
 setuptools.setup(
     name="ofrak_type",
-    version="2.2.0",
+    version="2.2.0rc0",
```

Comment:
Shouldn't this be 2.3.0rc0?

```suggestion
    version="2.3.0rc0",
```